src/data/meetings.js

```diff
@@ -19,9 +19,9 @@ export default {
     state: MeetingState.JOINED,
     showRoster: false,
     showSettings: false,
-    cameraID: 'cameraID1',
```

Comment:
Why did you remove these values?

Reply:
For testing only, I wanted to simulate when we have nothing selected for cameraID, microphoneID and speakerID
I will reset to the old values